src/main/kotlin/no/nav/dagpenger/soknad/orkestrator/behov/løsere/PermittertGrensearbeiderBehovløser.kt

```diff
@@ -0,0 +1,41 @@
+package no.nav.dagpenger.soknad.orkestrator.behov.løsere
+
+import com.github.navikt.tbd_libs.rapids_and_rivers_api.RapidsConnection
+import no.nav.dagpenger.soknad.orkestrator.behov.Behovløser
+import no.nav.dagpenger.soknad.orkestrator.behov.BehovløserFactory.Behov.PermittertGrensearbeider
+import no.nav.dagpenger.soknad.orkestrator.behov.Behovmelding
+import no.nav.dagpenger.soknad.orkestrator.quizOpplysning.db.QuizOpplysningRepository
+import java.util.UUID
+
+class PermittertGrensearbeiderBehovløser(
+    rapidsConnection: RapidsConnection,
+    opplysningRepository: QuizOpplysningRepository,
+) : Behovløser(rapidsConnection, opplysningRepository) {
+    override val behov = PermittertGrensearbeider.name
+    override val beskrivendeId
+        get() = throw NotImplementedError("Overrider løs() og trenger ikke beskrivendeId")
+
+    companion object {
+        val reistTilbakeEnGangEllerMerBeskrivendeId = "faktum.reist-tilbake-en-gang-eller-mer"
+        val reistITaktMedRotasjonBeskrivendeId = "faktum.reist-i-takt-med-rotasjon"
+    }
```

Comment:
Jeg gjorde dette likt som for `BarnetilleggBehovLøser`, som også henter fra to forskjellige faktum.

Reply:
De beskrivendeId'ene brukes i testen, men det er jo ikke sikkert at de trenger å være i companion object likevel.